UnityProject/Assets/Scripts/Util/Independent/FluentRichText/Styles/CommonValidations.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Util.Independent.FluentRichText.Styles
+{
+	public static class CommonValidations
+	{
+		public static bool IsValidPixelValue(string value)
+		{
+			return Regex.IsMatch(value, @"^-?\+?\d+(\.\d+)?$");
+		}
+
+		public static  bool IsValidFontUnitValue(string value)
+		{
+			return Regex.IsMatch(value, @"^-?\+?\d+(\.\d+)?em$");
+		}
+
+		public static bool IsValidPercentageValue(string value)
+		{
+			return Regex.IsMatch(value, @"^-?\+?\d+(\.\d+)?%$");
+		}
+
+		public static bool IsValidHexColor(string hexColor)
+		{
+			return Regex.IsMatch(hexColor, @"^#([a-fA-F0-9]{6}|[a-fA-F0-9]{8})$");
```

Comment:
My understanding is three or four characters like #fff should also be considered valid - or does TMP not support that?

Reply:
I'm not sure, I based all the implementation on the one site that has the TMP formatting documentation